spec/requests/static_pages_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe "StaticPages", type: :request do
+  describe "#home" do
+    it "should have the title Home" do
+      visit '/static_pages/home'
+
+      expect(page).to have_title('Expenses | Home')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.